alacritty_terminal/src/index.rs

```diff
@@ -123,6 +123,12 @@ impl Linear {
     pub fn from_point(columns: Column, point: Point<usize>) -> Self {
         Linear(point.line * columns.0 + point.col.0)
     }
+
+    pub fn to_point(self, columns: Column) -> Point<Line> {
```

Comment:
Take a look on rust's `to_vec` and `into_vec` methods (on slice) and refactor to follow one of them.